deps/rabbitmq_mqtt/src/rabbit_mqtt_processor.erl

```diff
@@ -87,6 +87,14 @@
 
 -opaque state() :: #state{}.
 
+-type info_commands() :: host | port | peer_host | peer_port
+    | connected_at | ssl_login_name | vhost | user_who_performed_action | user
+    | clean_sess | will_msg | retainer_pid | exchange | prefetch
+    | messages_unconfirmed | messages_unacknowledged
+    | node
+    | client_id | client_properties
+    | channel_max | frame_max | auth_mechanism.
+
```

Comment:
I would delete this whole block and instead simply use `rabbit_types:info_key()` for the function spec in `rabbit_mqtt_processor:info/1` because this is a long block and doesn't add much benefit from my point of view.